src/code/z_quake.c

```diff
@@ -40,230 +33,256 @@ typedef enum {
     /* 2 */ DISTORTION_SETUP
 } DistortionState;
 
-QuakeRequest sQuakeRequest[4];
+QuakeRequest sQuakeRequests[4];
 DistortionRequest sDistortionRequest;
 
-static s16 sIsQuakeInitialized = true;
+static s16 sQuakeUnused = 1;
```

Comment:
`sIsQuakeInitialized` isn't really a good name because it's set to true when Quake is NOT initialized. In fact, it's always set to 1, it's just that it's also set to 1 in `Init`. But the variable itself doesn't indicate if it's initialized or not

Reply:
Right, it's not a leftover or anything?

Reply:
From OoT, it's the exact same variable used in the exact same unused way. So there's not really any way of knowing its purpose unless its from an earlier OoT version